test/helper.js

```diff
@@ -236,16 +236,18 @@ module.exports.payloadMethod = function (method, t) {
         t.strictEqual(response.statusCode, 400)
       })
 
-      sget({
-        method: upMethod,
-        url: 'http://localhost:' + fastify.server.address().port,
-        body: '',
-        headers: { 'Content-Type': 'application/json' },
-        timeout: 500
-      }, (err, response, body) => {
-        t.error(err)
-        t.strictEqual(response.statusCode, 400)
-      })
+      if (upMethod !== 'OPTIONS') {
+        sget({
+          method: upMethod,
+          url: 'http://localhost:' + fastify.server.address().port,
+          body: '',
+          headers: { 'Content-Type': 'application/json' },
```

Comment:
To avoid the separate case for the OPTIONS request, you could remove the `body: '',` line and add a `'Content-Length': '0'` header.